src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
I still don't know *exactly* why this one uses ECMA (looking). I do know that more libraries will be added in the future that also use ECMA.